src/backend/marsha/core/lti/utils.py

```diff
@@ -9,6 +9,57 @@ class PortabilityError(Exception):
     """An error raised when trying to access a resource that is not portable."""
 
 
+def get_resources(model, lti):
+    """Get resources available to a LTI request.
+
+    This function is generic and will use the `model` argument to retrieve resources.
```

Comment:
The description does not reflect what you are doing

Reply:
test